### PR TITLE
added highway=emergency_access_point

### LINKED ISSUE #20: Potential adding a preset for highway=emergency_access_point
wiki: https://wiki.openstreetmap.org/wiki/Tag:highway%3Demergency_access_point

Seems approved:
https://wiki.openstreetmap.org/wiki/Proposed_features/emergency_access_point
and is well distributed:
https://taginfo.openstreetmap.org/tags/highway=emergency_access_point
https://taginfo.openstreetmap.org/tags/highway=emergency_access_point#map

Has quite constant organic growth:
https://taghistory.raifer.tech/#***/highway/emergency_access_point

Fields:
`ref`, (not sure about fields `emergency_telephone_code` and `operator` yet)

### PR DESCRIPTION
**Pull request by tiptoptom**
I tried to make a PR for highway=emergency_access_point. This is my first PR here so I hope, everything is fine. And if not forgive me =)

closes #20

### CODE REVIEW

**Review by tyrasd — CHANGES_REQUESTED** [review]
Hi. Thanks for your pull request. It definitely makes sense to include a preset for this feature. :+1: 

See the comments below for a few tweaks to be made before the preset can be merged into the index.

File: data/presets/highway/emergency_access_point.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "operator",
+        "name"
+    ],
+    "tags": {
+        "highway": "emergency_access_point"
+    },
+    "terms": [
+        "Highway Emergency Access Point"
+    ],
+    "name": "Highway Emergency Access Point"
```
[the remark this answers is not in the document]
Comment (tiptoptom) [reply data/presets/highway/emergency_access_point.json:19]: But for hiking trails etc. I think emergency=access_point is more intended. highway=emergency_access_point refers I think from the tag to a road.

[the remark this answers is not in the document]
Comment (tyrasd) [reply data/presets/highway/emergency_access_point.json]: Hm… the [wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Demergency_access_point) doesn't mention that `highway=emergency_access_point` is limited to roadside rescue points. On the contrary, it seems to indicate that off-road rescue points are the main intended purpose behind the tag:

> For areas which are hard to access, like woods or waterways, some countries offer a network of emergency access points.

Also, most examples I could find are along hiking paths, see https://overpass-turbo.eu/s/1jv1 to check yourself.

[the remark this answers is not in the document]
Comment (tiptoptom) [reply data/presets/highway/emergency_access_point.json]: Ok, you may be right. I just wanted to create a demarcation to emergency=access_point …
But that doesn't have to be right now.

**Review by tyrasd — APPROVED** [review]

**Pull request merged**